src/tests/ftest/osa/online_extend.py

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/python
 """
-  (C) Copyright 2020-2021 Intel Corporation.
+  (C) Copyright 2020-2022 Intel Corporation.
 
   SPDX-License-Identifier: BSD-2-Clause-Patent
 """
 import time
 import threading
 
-from test_utils_pool import TestPool
+from test_utils_pool import add_pool
```

Comment:
(pylint-syntax-error) Cannot import 'test_utils_pool' due to syntax error 'invalid syntax (<unknown>, line 53)'